pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -795,7 +795,11 @@ cdef class _Timedelta(timedelta):
 
     @property
     def resolution(self):
-        """ return a string representing the lowest resolution that we have """
+        """
+        Return a string representing the lowest resolution that we have.
```

Comment:
Instead of `lowest resolution that we have`, I think I'd be clearer if it was `lowest resolution of the Timedelta instance`